### PR TITLE
setup(cypress): add cypress dependencies

### PR DESCRIPTION
**Pull request by isaRosolen**
This PR installs the cypress dependencies

### CODE REVIEW

**Review by tamaracvorovic-work — CHANGES_REQUESTED** [review]
Hi Isabela, you forgot to add .gitignore file with node_modules, so please add that (you commited over 5000 files). 
Also in test, you said that something (you didn't define which element) contains text. The goal is to get exact element and compare expected text with text in DOM.
And it will be nice to add few sentences in README.md which will be explanation what is done.
Please make that changes. :)

File: cypress/e2e/spec.cy.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe('Open the website', () => {
+  it('passes', () => {
+    cy.visit('http://work.co/')
+
+    cy.contains('We design and ship digital products that transform companies.')
```
Comment (stefanworkco) [inline cypress/e2e/spec.cy.js]: `cy.contains` only finds the element containing that text - [doc](https://docs.cypress.io/api/commands/contains).
For assertion we need to use should - [doc](https://docs.cypress.io/api/commands/should#Syntax)

**Review by stefanworkco — CHANGES_REQUESTED** [review]
Like Tamara said - node_modules should be in gitignore 
After that, try fixing it like this: [link](https://stackoverflow.com/questions/61666908/how-to-remove-node-modules-from-the-pull-request#:~:text=You%20need%20to%20first%20delete,this%20won't%20be%20tracked.)

File: cypress/e2e/spec.cy.js (hunk 1 above)
Comment (stefanworkco) [reply cypress/e2e/spec.cy.js]: @IsabelaRosolen  still requires this change

Comment (isaRosolen) [reply cypress/e2e/spec.cy.js]: Thank you, @stefanworkco. Just added the changes.

**Comment by stefanworkco** [comment]
The tests work, but few things would be good to improve:
- update README.md with just desc. and commands for running the tests( something like: "To install dependencies run `npm install` ; To open test runner run 'npx cypress open' ") - I don't think you need the steps of the challenge in README
- cy.contains('We design and ship digital products that transform companies.').should('be.visible') - instead of using cy.contains, use cy.get (and for selector use 'data-test-id' attribute). Also, use `.should('have.text', expected_text)` instead of `be.visible`
- replace `'passes'` with some better description of the test step (inside the it()) 
- I'm pretty sure that this package-lock.json you added has a lot of extra (unnecessary) stuff since the lock file for cypress only has around 1350 lines, and this one is 3170 lines file... ( Try deleting the lock file in local, then running `npm install` again and push that newly generated package-lock.json)

**Review by stefanworkco — APPROVED** [review]
Awesome 🔝

**Review by tamaracvorovic-work — APPROVED** [review]
This looks great! :tada:

**Pull request merged**